ovirt-engine-hyper-upgrade.spec.in

```diff
@@ -76,6 +76,10 @@ make %{?_smp_mflags}
 %install
 make %{?_smp_mflags} install DESTDIR="%{buildroot}"
 
+%post
+ln -sf %{_sbindir}/ovirt-engine-hyper-upgrade %{_sbindir}/engine-hyper-upgrade
+ln -sf %{_mandir}/man8/ovirt-engine-hyper-upgrade.* %{_mandir}/man8/engine-hyper-upgrade.8.gz
```

Comment:
Added because of engine-setup. @didib @sandrobonazzola please let me know if you guys want a different name. Having 'ovirt-'engine-hyper-upgade because of upstream and engine-hyper-upgrade because of downstream is fine for me. On the other hand, the tool is downstream only but lives upstream.